pkg/make/builder.go

```diff
@@ -575,7 +575,7 @@ func (b *Builder) BuildCmd(src, appName string) func(context.Context) error {
 
 		// Tell github where we're at
 		if b.githubActionOutput {
-			fmt.Printf("::set-output name=binary::%s\n", output)
+			fmt.Printf("name=binary::%s >> $GITHUB_OUTPUT\n", output)
```

Comment:
The old syntax, was parsed from stdout. the new syntax is written into a file. 

So this needs to to have file open/append with `fmt.Fprintf(f, "binary=%s\n", output)`

Reply:
🤦 hah, yep!

Reply:
Based on the CI checks I think this is fixed now, but I haven't yet figured out where the output is and how to verify it.